.github/workflows/scorecards.yml

```diff
@@ -55,6 +66,10 @@ jobs:
           #     of the value entered here.
           publish_results: true
 
+      - name: Add Attestation (Sarif results)
+        run: |
+          chainloop attestation add --name sarif-results --value results.sarif
```

Comment:
That's what I would like to do now :D

Reply:
how are you going to test it? if you plan to do it as part of `PR hook` you'll need to create the PR in the same repo, not in a fork (because of security leak prevention). Do you want to merge it?

Reply:
I will then close this PR and perform the changes on the main repo so I can trigger it on the PR.

Reply:
ok